src/Types/Input/NullableInputType.hack

```diff
@@ -36,4 +34,13 @@ final class NullableInputType<TInner as nonnull>
     public function assertValidVariableValue(mixed $value): ?TInner {
         return $value is null ? null : $this->inner_type->assertValidVariableValue($value);
     }
+
+    <<__Override>>
+    final public function introspect(): GraphQL\Introspection\__Type {
+        $inner = $this->inner_type->introspect();
+        while ($inner is GraphQL\Introspection\NonNullableType) {
```

Comment:
This is a little hard to follow: if we're introspecting a nullable type, drill down till the first type that isn't nonnullable.

Reply:
There should never be more than but since we don't statically enforce it I thought this would be safer.